.mailmap

```diff
@@ -1700,6 +1701,7 @@ carstimon <carstimon@gmail.com>
 ck Lux <lux.r.ck@gmail.com>
 cocolato <haiizhu@outlook.com> Hai Zhu <35182391+cocolato@users.noreply.github.com>
 codecruisader <nnisarg55@gmail.com>
+cp50 <chrispaul68002@gmail.com>
```

Comment:
This line should be removed now that the other is added.

I'm not sure why that doesn't throw an error.